packages/mux-player/src/index.ts

```diff
@@ -339,11 +339,11 @@ class MuxPlayerElement extends VideoApiElement {
   }
 
   get forwardSeekOffset() {
-    return this.getAttribute(PlayerAttributes.FORWARD_SEEK_OFFSET);
+    return this.getAttribute(PlayerAttributes.FORWARD_SEEK_OFFSET) ?? 10;
```

Comment:
nit (non-blocking): we should be consistent with either treating these as strings or numbers. I.e. either:
```js
return this.getAttribute(PlayerAttributes.FORWARD_SEEK_OFFSET) ?? '10';
```
or
```js
if (!this.hasAttribute(PlayerAttributes.FORWARD_SEEK_OFFSET) return 10;
return +this.getAttribute(PlayerAttributes.FORWARD_SEEK_OFFSET);
```
(or similar)